helm/guppy/templates/guppy_config.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 data:
   guppy_config.json: |
       {
+        "allowRefresh": {{ .Values.allowRefresh }},
```

Comment:
this isn't a current config that's used anywhere right? This is in anticipation of the [feature/refresh](https://github.com/uc-cdis/guppy/pull/273) improvements?

Reply:
Good catch. This is being used here https://github.com/uc-cdis/guppy/pull/273 . Assuming this gets merged, we will want to use the new feature that it brings, the ability the refresh guppy as part of the ETL stack without having to do it manually.